README.md

```diff
@@ -25,6 +25,54 @@ Don't hesitate to make a discussion and we will try to answer as fast as we can.
 
 You know C# ? You can try picking an issue up and make a pull request and we will review it.
 
+## Developping maimai
+
+The following items are required to be installed on your computer in order to develop maimai:
+
+- The [.NET 5 SDK](https://dotnet.microsoft.com/en-us/download/visual-studio-sdks) or higher
```

Comment:
```suggestion
- The [.NET 6 SDK](https://dotnet.microsoft.com/en-us/download) or higher
```

We just move to .NET 6 and the link to downlaod is going to Visual Studio SDKs, not .NET SDKs so I just edit it.